mod-source-record-manager-server/src/main/java/org/folio/services/journal/JournalUtil.java

```diff
@@ -88,8 +88,8 @@ public static JournalRecord buildJournalRecordByEvent(DataImportEventPayload eve
             }
             journalRecord.setHoldingsId(entityJson.getString("holdingsRecordId"));
           }
-          if(entityType == PO_LINE){
-            journalRecord.setOrderId(entityJson.getString("orderId"));
+          if (entityType == PO_LINE){
+            journalRecord.setOrderId(entityJson.getString("purchaseOrderId"));
```

Comment:
@RomanChernetskyi  Do we have "purchaseOrderId"-field inside entityJson ?

Reply:
Yes, we have